src/DssSpell.sol

```diff
@@ -65,21 +113,266 @@ contract SpellAction {
 
     function execute() external limited {
         address MCD_VAT      = CHANGELOG.getAddress("MCD_VAT");
+        address MCD_CAT      = CHANGELOG.getAddress("MCD_CAT");
+        address MCD_SPOT     = CHANGELOG.getAddress("MCD_SPOT");
+        address MCD_JUG      = CHANGELOG.getAddress("MCD_JUG");
+        address MCD_END      = CHANGELOG.getAddress("MCD_END");
+        address MCD_VOW      = CHANGELOG.getAddress("MCD_VOW");
+        address FLIPPER_MOM  = CHANGELOG.getAddress("FLIPPER_MOM");
+        address OSM_MOM      = CHANGELOG.getAddress("OSM_MOM");
+        address MCD_JOIN_DAI = CHANGELOG.getAddress("MCD_JOIN_DAI");
+        address ILK_REGISTRY = CHANGELOG.getAddress("ILK_REGISTRY");
+
+        // Adjust Debt Ceiling Parameters - January 18, 2021
+        // https://vote.makerdao.com/polling/QmQtn7UY#poll-detail - LINK-A
+        // https://vote.makerdao.com/polling/QmSCLfXN#poll-detail - MANA-A
+        // https://vote.makerdao.com/polling/QmW4ei2M#poll-detail - BAT-A
+        // https://vote.makerdao.com/polling/QmXTGwq4#poll-detail - TUSD-A
+        // https://vote.makerdao.com/polling/QmfTU85J#poll-detail - PSM-USDC-A [ December 14, 2020 ]
 
         // Set the global debt ceiling
-        // + 260 M for ETH-A
-        // + 50 M for WBTC-A
+        // + 10 M for LINK-A
+        // + 750 K for MANA-A [ Note: Units ]
+        // - 8 M for BAT-A
+        // - 135 M for TUSD-A
+        // + 470 M for PSM-USDC-A [ Lerp End Amount ]
+        // TODO: WBTC-ETH UNI LP
+        // TODO: USDC-ETH UNI LP
+        // VatAbstract(MCD_VAT).file("Line",
+        //     VatAbstract(MCD_VAT).Line()
+        //     + 10 * MILLION * RAD
+        //     + 750 * THOUSAND * RAD
+        //     - 8 * MILLION * RAD
+        //     - 135 * MILLION * RAD
+        //     + 470 * MILLION * RAD
+        // );
+        // TODO: fix this, this includes 6M for the two uni lp tokens
         VatAbstract(MCD_VAT).file("Line",
             VatAbstract(MCD_VAT).Line()
-            + 260 * MILLION * RAD
-            + 50 * MILLION * RAD
+            + 337750 * THOUSAND * RAD
```

Comment:
Wouldn't be better to just do `+ 343_750_000 * RAD` in this case?

Reply:
Yeah good call.